tableauserverclient/server/request_options.py

```diff
@@ -117,51 +156,47 @@ class Direction:
         Desc = "desc"
         Asc = "asc"
 
-    def __init__(self, pagenumber=1, pagesize=None):
-        self.pagenumber = pagenumber
-        self.pagesize = pagesize or config.PAGE_SIZE
-        self.sort = set()
-        self.filter = set()
 
-        # This is private until we expand all of our parsers to handle the extra fields
-        self._all_fields = False
+"""
+These options can be used by methods that are fetching data exported from a specific content item
+"""
 
-    def page_size(self, page_size):
-        self.pagesize = page_size
-        return self
 
-    def page_number(self, page_number):
-        self.pagenumber = page_number
-        return self
+class _DataExportOptions(RequestOptionsBase):
+    def __init__(self, maxage: int = -1):
+        super().__init__()
+        self.view_filters: list[tuple[str, str]] = []
+        self.view_parameters: list[tuple[str, str]] = []
+        self.max_age: Optional[int] = maxage
+        self._language: Optional[str] = None
 
-    def get_query_params(self):
-        params = {}
-        if self.pagenumber:
-            params["pageNumber"] = self.pagenumber
-        if self.pagesize:
-            params["pageSize"] = self.pagesize
-        if len(self.sort) > 0:
-            sort_options = (str(sort_item) for sort_item in self.sort)
-            ordered_sort_options = sorted(sort_options)
-            params["sort"] = ",".join(ordered_sort_options)
-        if len(self.filter) > 0:
-            filter_options = (str(filter_item) for filter_item in self.filter)
-            ordered_filter_options = sorted(filter_options)
-            params["filter"] = ",".join(ordered_filter_options)
-        if self._all_fields:
-            params["fields"] = "_all_"
-        return params
+    """
+    This setting will affect the contents of the workbook as they are exported.
+    Valid language values are tableau-supported languages like de, es, en
+    If no locale is specified, the default locale for that language will be used
+    """
 
+    def language(self, value):
+        self._language = value
 
-class _FilterOptionsBase(RequestOptionsBase):
-    """Provide a basic implementation of adding view filters to the url"""
+    @property
+    def max_age(self):
+        return self._max_age
 
-    def __init__(self):
-        self.view_filters = []
-        self.view_parameters = []
+    @max_age.setter
+    @property_is_int(range=(0, 240), allowed=[-1])
+    def max_age(self, value):
+        self._max_age = value
 
     def get_query_params(self):
-        raise NotImplementedError()
+        params = {}
+        if self.max_age != -1:
+            params["maxAge"] = self.max_age
+        if self.language:
```

Comment:
Should this be `self._language`? I only see a setter for `language`, not a getter.